.buildkite/run_benchmark.yml

```diff
@@ -15,7 +15,7 @@ steps:
             - BUILDKITE_S3_SECRET_ACCESS_KEY="U2FsdGVkX1+LWh1yX7LsMBlecEJLc08eJrgOhurhd47CY1/jS3wCGVCQmS1t6f2j70spBoFdfc9kn2naj8HH5A==" 
             - BUILDKITE_S3_DEFAULT_REGION="U2FsdGVkX18ccoE9FmtkwsCm1x0MLMBlN/FLcAyKkY4="
       - JuliaCI/julia#v1:
-          version: 1.9
+          version: 1.10
```

Comment:
This might need to be update to `version: "1.10"`.

Reply:
ah that does it. Thanks!